nixos/modules/services/misc/nix-gc.nix

```
@@ -19,7 +19,7 @@ in
       };
 
       dates = lib.mkOption {
-        type = lib.types.singleLineStr;
+        type = with lib.types; either singleLineStr (listOf str);
         default = "03:15";
```

Can we do something like

```suggestion
        apply = lib.toList;
        default = "03:15";
```